docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,96 @@
+
+`AzureClient` can be connected to a `Azure local service (Tinylicious)` instance by passing in the tenant ID as `local`, the orderer and storage URLs  pointing to the Tinylicious instance on the default values of `http://localhost:7070`. You can use `InsecureTokenProvider` for token resolution while running the service locally for development purposes.
+
+In the below code snippet, `AzureClient` is pointing to Tinylicious service. This is providing you with an added feature to work with both the local Tinylicious service and the deployed Azure Fluid Relay service. `TinyliciousClient` only works with local Tinylicous service.
+
+```typescript
+import { AzureClient, AzureConnectionConfig } from "@fluidframework/azure-client";
+import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
+
+const user = {
+    id: "UserId",
+    name: "Test User",
+};
+
+const config: AzureConnectionConfig = {
+    tenantId: "local",
+    tokenProvider: new InsecureTokenProvider("fooBar", user),
+    // if you're running Tinylicious on a non-default port, you'll need change these URLs
+    orderer: "http://localhost:7070",
+    storage: "http://localhost:7070",
+};
+
+const clientProps = {
+  connection: config,
+}
+
+const client = new AzureClient(clientProps);
+```
+
+To launch the local Tinylicious service instance, run `npx @fluidframework/azure-local-service@latest` from your terminal window.
+
 ## Using custom domains with Tinylicious
+
+You can use `ngrok` which lets you expose port on your local machine to the internet. Ngrok gives you a random hostname for each tunnel. This tool enables you to attach domains/subdomains against those tunnels.
+
+1. Navigate to [ngrok](https://ngrok.com/) and Sign-up. There would be an `AUTH_TOKEN` generated on signing up.
+
+2. Download [ngrok](https://ngrok.com/download) and unzip the folder
+
+3. Connect to your account by running the following command,
+
+```sh
+ngrok authtoken AUTH_TOKEN
+```
+
+Running this command will add your authtoken to the default ngrok.yml configuration file.
+
+4. Run Tinylicious service locally
+
+```sh
+npx tinylicious@latest
+```
+
+5. Fire ngrok. By default, Tinylicious is running on port 7070, so the `PORT_NUMBER` in the below command would be `7070`. If you are running againt a non-default port, the `PORT_NUMBER` would vary.
+
+```sh
+ngrok http PORT_NUMBER
+```
+
+After running this command, you will see the `Forwarding` URL in your terminal, which can be used to access Tinylicious by replacing the `orderer` and `storage` URLs in the `AzureConnectionConfig`.
+
+If you are using the `Free` subscription of `ngrok`, there would random hexadecimal names generated to the HTTP tunnel opened for you. If you want to specify your subdomain, you would have to upgrade to their [paid plans](https://dashboard.ngrok.com/billing/plan). Once upgraded, you can run the below command by specifying your custom domain name and port number on which your local Tinylicious service is running.
```

Comment:
```suggestion
If your ngrok account includes the capability to set custom domains or subdomains, you can use the following command to use a custom domain instead of a randomly-generated one.
```